rtbkit/core/router/filters/static_filters.cc

```diff
@@ -82,6 +82,17 @@ applyExchangeFilter(FilterState& state, const ConfigSet& result) const
 
 void
 SegmentsFilter::
+fillFilterReasons(FilterState& state, ConfigSet& beforeFilt,
+                  ConfigSet& afterFilt, const std::string & segment) const {
+
+    // Some Magic to get all the filtered out configs by this segment.
+    FilterState::FilterReasons& reasons = state.getFilterReasons();
+    reasons[segment] = beforeFilt ^ (beforeFilt & afterFilt);
```

Comment:
Yes, you are right, i wil change it. 
The reason why i did that is because i look into this : https://github.com/rtbkit/rtbkit/blob/master/rtbkit/core/router/filters/static_filters.cc#L64.